pulp.spec

```diff
@@ -390,6 +390,8 @@ Pulp provides replication, access, and accounting for software repositories.
 %{_var}/lib/%{name}/uploads
 %dir %{_var}/log/%{name}
 %{_var}/www/pub
+%{_var}/cache/%{name}/
```

Comment:
This looks right, but what are the contents of this directory? I looked on an installed Pulp system, and this folder didn't even exist.